v3/src/components/graph/register-adornment-types.ts

```diff
@@ -0,0 +1,9 @@
+import "./adornments/movable-line-registration"
+
+const gAdornmentRegistration: Record<string, () => void> = {
+  "MovableLine": () => import(/* webpackChunkName: "MovableLine" */"./adornments/movable-line-registration")
+};
+
+export function registerAdornmentTypes(adornmentTypeIds: string[]) {
+  return Promise.all(adornmentTypeIds.map(id => gAdornmentRegistration[id]?.()))
+}
```

Comment:
The point of using dynamic imports with tiles is that in CLUE currently (and potentially down the road in CODAP), Webpack is configured to package individual tiles into their own bundles and so, for instance, all of the code for the Dataflow tile can be included/excluded depending on the unit configuration. Dynamic loading isn't a concern for individual adornments, so the dynamic imports approach isn't necessary. For CODAP's purposes, all that is required is that the desired adornment registration files be imported.
```suggestion
```